scripts/us_census/pep/Population_Estimate_by_ASR/county_1970_1979.py

```diff
@@ -0,0 +1,101 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+'''
+This Python Script is
+for County Level Data
+1970-1979
+'''
+import os
+import json
+import pandas as pd
+
+
+def county1970():
```

Comment:
There is a lot of duplicate code in these country_*.py and national_*.py and state_*.py files. Is it not possible to have a common superclass (or util functions) which allow you to abstract the common processing which can be used by these subclasses/functions? My worry is that now, if one needs to make a small change which is common to many/all of these processors, it will need to be manually done in all of them. Ideally, we'd find the one helper function which does this and know it will be used everywhere...

Reply:
Ok, thanks!